addon/components/paper-chips.js

```diff
@@ -170,7 +171,7 @@ export default Component.extend({
 
     noUnselected(old, event) {
       if (['Backspace', 'Delete', 'Del', 'ArrowLeft', 'Left', 'ArrowRight', 'Right'].includes(event.key)) {
-        this.sendAction('keyDown', event);
+        invokeAction(this, 'keyDown', item);
```

Comment:
this should be `event` instead of `item`

Reply:
Exactly. This is the error on travis.
Linting tests are useful after all, huh.